cmd/tsdb/main.go

```diff
@@ -55,6 +57,13 @@ func main() {
 		analyzeLimit         = analyzeCmd.Flag("limit", "how many items to show in each list").Default("20").Int()
 	)
 
+	options := &tsdb.Options{
+		WALSegmentSize:    wal.DefaultSegmentSize,
+		RetentionDuration: 15 * 24 * 60 * 60 * 1000, // 15 days in milliseconds
+		BlockRanges:       tsdb.ExponentialBlockRanges(int64(2*time.Hour)/1e6, 3, 5),
+		NoLockfile:        false,
```

Comment:
since this is set as a global option I expect this to be set here
```suggestion
		NoLockfile:        *noLock,
```